bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/graphics/ImageLoader.java

```diff
@@ -168,25 +168,17 @@ public ImageData[] load(InputStream stream) {
  *    <li>ERROR_NULL_ARGUMENT - if the file name is null</li>
  * </ul>
  * @exception SWTException <ul>
- *    <li>ERROR_IO - if an IO error occurs while reading from the file</li>
+ *    <li>ERROR_IO - if an IO error occurs while reading the file</li>
```

Comment:
This adjustment of the exception description is fine in my opinion, but I also think the previous text was ok.
However we have now an inconsistency with the `ImageLoader` class for Windows were the from is still present.
Plus the `load(InputStream)` method has a similar documentation for `ERROR_IO`, were we should stay consistent too.
So if you want to remove the `from`, please also remove it for Windows and the `load(InputStream)` method, or revert this this removal. As you prefer :)

Reply:
thank you, did not see it in time. :)

Reply:
No problem and you are welcome. :)